### ISSUE TITLE
Show what Level Pack(s) a song is in

### ISSUE THREAD

**Issue by cryptk** [the-expanse/SideQuest#139]
When going through the Custom Levels screen sorting songs into packs, it would be really nice if the song entries in the My Levels list would show what Level Pack(s) that song was in... this way I could easily see if a song was in a Level Pack or not already.

It would be really nice if there was an indicator that could quickly show that a song is not in any Level Pack... this would make finding those new downloads in the list fast and easy.  Perhaps you could also add a Sort option to move songs that aren't in a pack to the top of the list as well.

**Comment by retorquere** [comment]
I really need this -- I know javascript/typescript so I'm willing to give this a go. Where does the code live that manages the levels? I'm new to angular.

**Comment by shaneharris** [comment]
The latest version ( out soon ) has a unique button for pack songs, it also has sorting by recent and name and it also allows a multiselect option for loading in songs to packs quickly and easily. 
[video](https://cdn.discordapp.com/attachments/589189586190860288/591005214803230720/DesktopApp_19_06_2019_21_40_53_Trim.mp4)

I hope this works!

**Comment by cryptk** [comment]
@shaneharris this doesn't really resolve the issue... let me rephrase the root problem:

Lets say you have 10 packs... and 200 songs... I would like to be able to quickly tell what packs a given song is in.  Perhaps the song isn't in a pack, it would be nice to be able to tell, at a glance, which of my 200 songs are not in a pack... without having to scroll through all 10 packs for each of the 200 songs.

Adding some information about what packs a song is in would resolve this issue.  Perhaps adding a small icon with a number next to the song name showing how many packs it is in, and then on mouseover, you would get a tooltip that lists the actual pack names.
